build-docker-image/action.yml

```diff
@@ -43,6 +47,14 @@ inputs:
     description: Whether load docker image from build into local Docker Images
     required: false
     default: 'false'
+  qemu_image:
+    description: QEMU static binaries Docker image
+    required: false
+    default: 'tonistiigi/binfmt:latest'
```

Comment:
you'd probably want to use `tonistiigi/binfmt:master`, `latest` has been outdated for ~2 years.

Reply:
I'd go with `master` for now.
See also https://github.com/docker/setup-qemu-action?tab=readme-ov-file#about where they are using `master` instead of `latest` even though every piece of their own documentation points to `latest`.